### PR TITLE
517 upd stock report secondary unit

### PR DESCRIPTION
**Pull request by SokreamPhanqrl**

### CODE REVIEW

File: stock_report_secondary_unit/i18n/ja.po [hunk 1]
```diff
@@ -35,6 +35,11 @@ msgstr "引当済"
 msgid "Secondary Qty"
 msgstr "数量(副単位)"
 
+#. module: stock_report_secondary_unit
+#: model_terms:ir.ui.view,arch_db:stock_report_secondary_unit.view_stock_quant_tree
+msgid "Secondary Quantity"
+msgstr ""
```
Comment (yostashiro) [inline stock_report_secondary_unit/i18n/ja.po]: `数量(副単位)`

File: stock_view_adj_hls/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "description": """
+    """,
+    'version': '12.0.1.0.0',
+    "category": "Stock",
+    "website": "https://www.quartile.co/",
+    "author": "Quartile Limited",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+         'stock_report_secondary_unit',
```
Comment (yostashiro) [inline stock_view_adj_hls/__manifest__.py]: Why do we need this dependency?

**Review by yostashiro — CHANGES_REQUESTED** [review]
Can you please bump the version of stock_report_secondary_unit to `12.0.1.0.1`?

File: stock_view_adj_hls/__manifest__.py (hunk 2 above)
Comment (SokreamPhanqrl) [reply stock_view_adj_hls/__manifest__.py]: we need it to use the field removal_date.

**Review by yostashiro — APPROVED** [review]
LGTM! ;)

**Pull request merged**